packages/browser/src/core/user/index.ts

```diff
@@ -171,7 +176,11 @@ export class LocalStorage extends Store {
   }
 
   remove(key: string): void {
-    return localStorage.removeItem(key)
+    try {
+      return localStorage.removeItem(key)
+    } catch (err) {
+      console.warn(err)
```

Comment:
I think we should match the warnings we have for `set` and `get` here as well.

Reply:
^ to follow up, we probably want to create a function so we don't repeat ourselves and avoid duplicate strings -- they take up precious bytes.